homeassistant/components/xiaomi_miio/select.py

```diff
@@ -21,53 +25,120 @@
     CONF_DEVICE,
     CONF_FLOW_TYPE,
     DOMAIN,
-    FEATURE_SET_LED_BRIGHTNESS,
     KEY_COORDINATOR,
     KEY_DEVICE,
+    MODEL_AIRFRESH_T2017,
     MODEL_AIRFRESH_VA2,
-    MODEL_AIRPURIFIER_3C,
+    MODEL_AIRHUMIDIFIER_CA1,
+    MODEL_AIRHUMIDIFIER_CA4,
+    MODEL_AIRHUMIDIFIER_CB1,
+    MODEL_AIRHUMIDIFIER_V1,
+    MODEL_AIRPURIFIER_3,
+    MODEL_AIRPURIFIER_3H,
     MODEL_AIRPURIFIER_M1,
     MODEL_AIRPURIFIER_M2,
+    MODEL_AIRPURIFIER_PROH,
     MODEL_FAN_SA1,
     MODEL_FAN_V2,
     MODEL_FAN_V3,
     MODEL_FAN_ZA1,
     MODEL_FAN_ZA3,
     MODEL_FAN_ZA4,
-    MODELS_HUMIDIFIER_MIIO,
-    MODELS_HUMIDIFIER_MIOT,
-    MODELS_PURIFIER_MIOT,
 )
 from .device import XiaomiCoordinatedMiioEntity
 
+ATTR_DISPLAY_ORIENTATION = "display_orientation"
 ATTR_LED_BRIGHTNESS = "led_brightness"
-
-
-LED_BRIGHTNESS_MAP = {"Bright": 0, "Dim": 1, "Off": 2}
-LED_BRIGHTNESS_MAP_HUMIDIFIER_MIOT = {"Bright": 2, "Dim": 1, "Off": 0}
-LED_BRIGHTNESS_REVERSE_MAP = {val: key for key, val in LED_BRIGHTNESS_MAP.items()}
-LED_BRIGHTNESS_REVERSE_MAP_HUMIDIFIER_MIOT = {
-    val: key for key, val in LED_BRIGHTNESS_MAP_HUMIDIFIER_MIOT.items()
-}
+ATTR_LED_BRIGHTNESS_HUMIDIFIER_MIOT = "led_brightness_miot"
```

Comment:
Main reason why we have _miot because LedBrightness enum has different ordering for miot 
airhumidifier_miot:
```
class LedBrightness(enum.Enum):
    Off = 0
    Dim = 1
    Bright = 2
```

airpurifier:
```
class LedBrightness(enum.Enum):
    Bright = 0
    Dim = 1
    Off = 2
```

if python has some way to construct enum element `self._enum_class(self._options_map[attr_value]),` by key, say me, and I remove this options_map related things. I don`t like it :(

Reply:
Hi @rytilahti. 
Was avoided by using enum_class._member_map_ instead predefined options_map.  options_map now using if we want to change enum key to something more readable. Is this ok?